site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,384 @@
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <!-- Navbar with links -->
+      <div id="global-header-1.1" class="navbar-collapse collapse me-lg-auto global-header-1">
+        <ul class="navbar-nav">
+          <li class="nav-item"><a class="nav-link active" href="#">Discover</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Shop</a></li>
+        </ul>
+      </div>
+      <!-- Navbar with action icons -->
+      <div id="global-header-1.2" class="navbar-collapse collapse d-sm-flex global-header-1">
+        <ul class="navbar-nav flex-row">
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/5.2/assets/img/boosted-sprite.svg#search"></use>
+              </svg>
+              <span class="visually-hidden">Search</span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/5.2/assets/img/boosted-sprite.svg#buy"></use>
+              </svg>
+              <span class="visually-hidden">Basket</span>
+              <span class="position-relative align-self-start">
+                <span class="badge bg-info rounded-pill position-absolute top-0 fs-6 text-white translate-middle">
+                  1
+                  <span class="visually-hidden">shopping basket items</span>
+                </span>
+              </span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <img src="/docs/5.2/assets/img/navbar-contact.png" width="25" height="25" role="img" alt="User"
+                loading="lazy">
+              <span class="visually-hidden">My account</span>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+</header>
+<!-- Main -->
+<main class="bd-content order-1">
+  <div class="bg-white title-bar mb-2 mb-md-0">
+    <div class="container-xxl d-block">
+      <div class="row">
+        <div class="col-md-8 mx-auto">
+          <h1 class="display-1">Become a resseler</h1>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container-xxl pt-md-2 pb-md-4">
+    <div class="row">
+      <div class="col-md-8 mx-auto">
+        <p class="ll-sm h3 mb-4 pb-0 pb-md-2">You are a professional and wish to resell Orange products to your
+          customers?
+          <br>Please fill the form and we will get back to you shortly.
+        </p>
+        <form class="needs-validation mb-4 pb-4 pb-md-2" novalidate>
```

Comment:
In order to be more comprehensive to people, I think we can get all spacings inside this tag.

```suggestion
        <form class="needs-validation mb-5" novalidate>
```